arbitrator/tools/stylus_benchmark/src/generate_wats.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2021-2024, Offchain Labs, Inc.
+// For license information, see https://github.com/OffchainLabs/nitro/blob/master/LICENSE
+
+use std::fs::File;
+use std::io::Write;
+use std::path::PathBuf;
+
+fn generate_add_i32_wat(mut out_path: PathBuf) -> eyre::Result<()> {
+    let number_of_ops = 20_000_000;
```

Comment:
You are right :), the size of the program was impacting a lot benchmark performance.
The strategy that you mentioned is around 5 times faster than not using a loop at all.

Instead of using toggle_benchmark to specify a single code block to be benchmarked, I added start_benchmark and end_benchmark instructions, that enables to specify multiple execution blocks to be benchmarked.
I tried to only benchmark the execution block inside the loop, but calling start_benchmark/end_benchmark multiple times introduces a performance overhead, that is not worth when compared to having a single start_benchmark/end_benchmark block that includes the loop.

Thanks!!!